Makefile

```diff
@@ -135,9 +135,7 @@ PARAM1 := "-DNUM_BUILDING_JOBS=${COMPILE_JOBS} -DENABLE_PYBINDS=1 -S. -B${RELEAS
 PARAM2 := "--build ${RELEASE_BUILD_DIR} --parallel ${COMPILE_JOBS}"
 PARAM3 := "${RELEASE_BUILD_DIR}"
 
-.PHONY: pyvsag
```

Comment:
We need to keep the subcommand to build the pyvsag package easily from the command line